Gems/Atom/RPI/Code/Source/RPI.Public/ViewportContext.cpp

```diff
@@ -141,19 +141,19 @@ namespace AZ
             UpdatePipelineRefreshRate();
         }
 
-        float ViewportContext::GetTargetFramerate() const
+        float ViewportContext::GetTargetFrameRate() const
```

Comment:
suggestion: As this function is doing more than simply returning a value (a simpler getter) it might be worth renaming to `QueryTargetFrameRate` instead to give a hint that more work is required (could also be `[[nodiscard]]` to ensure it's not used incorrectly)

Reply:
I missed this - likely not going to get to this bit of feedback for this, round, apologies, but it's a good suggestion and I'm happy to pick it up in a follow-up pass.